src/scenario.jl

```diff
@@ -368,29 +366,41 @@ function run_model(domain::Domain, param_set::NamedDimsArray, corals::DataFrame,
 
     p.mb .= corals.mb_rate  # background mortality
     @set! p.k = max_cover  # max coral cover
-    @set! p.comp = sim_params.comp  # competition rate
 
     # Proportionally adjust initial cover (handles inappropriate initial conditions)
     proportional_adjustment!(Y_cover[1, :, :], cover_tmp, max_cover)
+    
+    # Define constant table location for seed values and seeded coral types
+    taxa_to_seed = [2,3,5]
+    taxa_names = ["seed_CA","seed_TA","seed_SM"]
```

Comment:
Get the seeded taxa names from `param_set`.

You tried to make things more generic so no sense in hardcoding more than you have to.

Reply:
Sure, so using `occursin()`, find everything with "seed_"? Or is there another way to do this?

Reply:
Yes, `occursin()` is one way. Not sure how performant it is but given it should only happen once per run I'm less concerned about it.

Later we could move this to somewhere else and set as a constant.